recipes/gradio-client/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "gradio-client" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gradio_client-{{ version }}.tar.gz
+  sha256: dc6479a119314aac0bbf6821da6e946df17f048cc571559379a89590618f7b5d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - hatchling
+    - hatch-requirements-txt
+    - hatch-fancy-pypi-readme >=22.5.0
+    - pip
+  run:
+    - python >=3.7
+    - fsspec
+    - huggingface_hub >=0.13.0
+    - packaging
+    - requests
+    - websockets
+
+test:
+  imports:
+    - gradio_client
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/gradio/
```

Comment:
This should probably be: https://pypi.org/project/gradio-client

Also consider adding a `dev_url: https://github.com/gradio-app/gradio`

Reply:
Thank you. I'll make the changes.